just_audio/example/pubspec.lock

```diff
@@ -571,5 +557,5 @@ packages:
     source: hosted
     version: "2.2.1"
 sdks:
-  dart: ">=2.10.0-110 <2.11.0"
+  dart: ">=2.12.0-0.0 <=2.12.0-179.0.dev"
```

Comment:
I have a separate `nnbd` branch which opts into null safety, so would rather keep `master` on the lower min SDK for now.